### DISCUSSION TITLE
Query Regarding Functionality Similar to ivy-restrict-to-matches

### DISCUSSION THREAD

**Discussion by t6ui** [minad/vertico#467] [Q&A] [answered]
I am an Emacs user currently exploring the Vertico ecosystem. One of the features I found extremely useful in Ivy was ivy-restrict-to-matches, which allows for restricting the list of candidates to the current matches and performing further narrowing from there.

I was wondering if there exists a known extension, package, or configuration that enables similar functionality within Vertico. Does Vertico support, or can it potentially support, a feature that would restrict candidate selections to matches that comply with the current search input, enabling subsequent narrowing of choices?

Best regards！

**Comment by minad** [comment, marked as answer]
See https://github.com/minad/vertico/wiki#restrict-the-set-of-candidates
